frontend/src/components/widgets/inputs/NumberInput.vue

```diff
@@ -240,13 +242,21 @@ export default defineComponent({
 			if (invalid) sanitized = this.value;
 
 			if (this.isInteger) sanitized = Math.round(sanitized);
-			if (typeof this.min === "number" && !Number.isNaN(this.min)) sanitized = Math.max(sanitized, this.min);
-			if (typeof this.max === "number" && !Number.isNaN(this.max)) sanitized = Math.min(sanitized, this.max);
+			sanitized = clamp(newValue, this.min, this.max);
 
 			if (!invalid) this.$emit("update:value", sanitized);
 
-			const roundingPower = 10 ** this.displayDecimalPlaces;
-			const displayValue = Math.round(sanitized * roundingPower) / roundingPower;
+			this.setText(sanitized);
+		},
+		setText(value: number) {
+			// Find the amount of digits on the left side of the number
+			// 10.25 == 2
+			// 1.23 == 3
+			// 0.23 == 0 - Reason for the slightly more complicated code
```

Comment:
These examples really don't make any sense to me. `10.25 -> 2` leads me to believe "left side digits" means left of the decimal point. `1.23` leads me to believe it means all the digits. `0.23` leads me to believe it means left of the decimal point. What does "left side digits" and "digits on the left side of the number" mean?

Reply:
I messed it up. It's supposed to say `1.23 == 1`

Reply:
Comment should also say left side of the **decimal point**.

Reply:
I think once CI passes this should be good to go

Reply:
Looks good. I was specifically referring to the first line of the comment, where it confusingly says "left side of the number" rather than "left side of the decimal" but your additions to the subsequent lines can probably clear up any confusion that causes. Feel free to fix or merge now, I gave my approval.